### PR TITLE
Add test rust command

### LINKED ISSUE #1906: Integrate rust test runner with python CLI
Users should be able to run rust test runner with python CLI, it can be implemented as separate command `protostar test-rust`

### PR DESCRIPTION
**Pull request by karol-bisztyga**
### Checklist
- [x] Relevant issue is linked
- [x] Docs updated/issue for docs created
- [x] Added relevant tests

Cairo PR https://github.com/software-mansion-labs/cairo/pull/78

### CODE REVIEW

**Review by piotmag769 — APPROVED** [review]

**Comment by MaksymilianDemitraszek** [comment]
Please not merge it right now

File: protostar/commands/cairo1_commands/test_rust_command.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                name="path",
+                description="A path to the file with tests that will be run",
+                type="str",
+                is_positional=True,
+            ),
+        ]
+
+    async def run(self, args: Namespace):
+        if args.path is None:
+            raise ProtostarException("No tests provided")
```
Comment (piotmag769) [inline protostar/commands/cairo1_commands/test_rust_command.py:37]:
```suggestion
                is_positional=True,
                is_required=True,
            ),
        ]

    async def run(self, args: Namespace):
```

File: pyrightconfig.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "ignore": ["./starknet_in_rust/", "./blockifier/"]
```
Comment (piotmag769) [inline pyrightconfig.json:2]: Do we plan on using starknet in rust?

**Comment by MaksymilianDemitraszek** [comment]
I think CLI is much closer to being merge than declare in rust right now. I don't think we should merge this one. Having additional bindings to python introduces additonal complexity and I prefer we avoided it.

**Comment by piotmag769** [comment]
Is this PR needed anymore or can we close it?
@karol-bisztyga @MaksymilianDemitraszek

**Comment by karol-bisztyga** [comment]
I'm going to close it once I come up with a replacement, I don't think it trashes the PR list that much and I use it as a ref so it's handy to have it as an open PR for me.

**Pull request closed without merging**